apps/heft/src/plugins/ApiExtractorPlugin/ApiExtractorPlugin.ts

```diff
@@ -36,28 +32,40 @@ interface IRunApiExtractorOptions {
   debugMode: boolean;
   watchMode: boolean;
   production: boolean;
+  apiExtractorJsonFilePath: string;
 }
 
 export class ApiExtractorPlugin implements IHeftPlugin {
   public readonly pluginName: string = PLUGIN_NAME;
 
   public apply(heftSession: HeftSession, heftConfiguration: HeftConfiguration): void {
     const { buildFolder } = heftConfiguration;
-    if (FileSystem.exists(path.join(buildFolder, CONFIG_FILE_LOCATION))) {
-      heftSession.hooks.build.tap(PLUGIN_NAME, (build: IBuildStageContext) => {
-        build.hooks.bundle.tap(PLUGIN_NAME, (bundle: IBundleSubstage) => {
-          bundle.hooks.run.tapPromise(PLUGIN_NAME, async () => {
+
+    heftSession.hooks.build.tap(PLUGIN_NAME, async (build: IBuildStageContext) => {
+      build.hooks.bundle.tap(PLUGIN_NAME, (bundle: IBundleSubstage) => {
+        bundle.hooks.run.tapPromise(PLUGIN_NAME, async () => {
+          // API Extractor provides an ExtractorConfig.tryLoadForFolder() API that will probe for api-extractor.json
+          // including support for rig.json.  However, Heft does not load the @microsoft/api-extractor package at all
+          // unless it sees a config/api-extractor.json file.  Thus we need to do our own lookup here.
+          const apiExtractorJsonFilePath:
+            | string
+            | undefined = await heftConfiguration.rigConfig.tryResolveConfigFilePathAsync(
```

Comment:
@iclanton this is my solution for the problem from your PR https://github.com/microsoft/rushstack/pull/2241